crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -646,15 +659,15 @@ pub enum OnExisting {
     /// files in the directory.
     Allow,
     /// Remove an existing directory.
-    Remove,
+    Remove(Option<OnExistingSource>),
```

Comment:
I might call this enum something like, `RemovableReason` instead? since it's scoped to the `Remove` variant not all `OnExisting` variants?